table/table.go

```diff
@@ -127,24 +145,24 @@ type Table interface {
 	// AllocAutoID allocates an auto_increment ID for a new row.
 	AllocAutoID(ctx sessionctx.Context) (int64, error)
 
-	// Allocator returns Allocator.
-	Allocator(ctx sessionctx.Context) autoid.Allocator
-
 	// RebaseAutoID rebases the auto_increment ID base.
 	// If allocIDs is true, it will allocate some IDs and save to the cache.
 	// If allocIDs is false, it will not allocate IDs.
 	RebaseAutoID(ctx sessionctx.Context, newBase int64, allocIDs bool) error
 
-	// Meta returns TableInfo.
-	Meta() *model.TableInfo
-
 	// Seek returns the handle greater or equal to h.
 	Seek(ctx sessionctx.Context, h int64) (handle int64, found bool, err error)
 
 	// Type returns the type of table
 	Type() Type
 }
 
+// PartitionTable inherits Table and it's a real Table, rather than a partition.
```

Comment:
Can not understand "it's a real Table, rather than a partition".

Reply:
How about this:

> PartitionTable is a Table, and it implements `GetPartition()` method.